### ISSUE TITLE
Role property for menuitem not applied to correct element (4.1.2)

### ISSUE THREAD

**Issue by Blackbaud-MattGregg** [blackbaud/skyux-popovers#89]
### Expected behavior - from RE NXT A11y audit in 2019

The ARIA based context menus are not implemented properly. There are some issues including:
- The role of menuitem should be applied to the element which is a keyboard event listener, that is the button element and not to the element containing the button element.

### Impact
The button menu does not announce proper role to the screen reader users. As a result, they might not be able to interact with the menu effectively.

**Comment by Blackbaud-TrevorBurch** [comment]
To change how this works will be a breaking change for consumers. We need to reevaluate this as a part of the next Sky UX major version.
